### PR TITLE
Revamped README and other .md files.  Added more clear breakdown betw…

### PR DESCRIPTION
**Pull request by jberkus**
…een build and contributing.  Added approval process to Contributing.

### CODE REVIEW

**Comment by jberkus** [comment]
@bproffitt @jzb please look over to see if this is an improvement.  Also, approval process we discussed merged in.

File: README.md [hunk 1]
```diff
@@ -1,90 +1,17 @@
...
-
-Simply add a new `gem 'some-gem-here'` line in the `Gemfile` and run
-`bundle install`
-
-
-## More info
-
-For more information, please check the excellent
-[Middleman documentation](https://middlemanapp.com/basics/install/).
+General discussion of the site often takes place on atomic-devel@projectatomic.io mailing list.  To contact an individual in charge of the site, please see MAINTAINERS.md.
```
Comment (mscherer) [inline README.md]: I think a link to the web interface of mailman would be useful to find the list.

File: BUILD.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+This website is built on Middleman, a static site generator. See below for
+quick instructions:
+
+To get started, you need to have Ruby and Ruby Gems installed, as well
+as "bundler".
+
+
+## Initial setup
+
+On an rpm based distribution:
```
Comment (jzb) [inline BUILD.md]: On Fedora. I'm not sure this works on CentOS, and I'm really not sure about SUSE, etc.

File: BUILD.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+
+## Updating
+
+When there are new gems in `Gemfile`, just run `bundle` again.
+
+
+## Customizing your site
+
+The site can be easily customized by editing `data/site.yml`.
```
Comment (jzb) [inline BUILD.md]: s/easily customized/customized/

File: BUILD.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+
+## Adding a Post
+
+To add a post to the community blog (or any blog managed by middleman) use:
+
+```
+bundle exec middleman article TITLE
```
Comment (jzb) [inline BUILD.md]: Note that there's also an admin web-based interface people can use. If they're going to run the site might as well get the full benefit of all that. Just go to http://0.0.0.0:4567/admin/.

File: BUILD.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+## Deploying
+
+### Setting up deployment
+
+FIXME: Right now, please reference <data/site.yml>
+
+### Actual deployment
+
+After copying your public key to the remote server and configuring your
+site in <data/site.yml>, deployment is one simple command:
```
Comment (jzb) [inline BUILD.md]: s/one simple/one/

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,20 +1,40 @@
----
-# Project Atomic Site Style Guide
+# Contributing
+
+We want your contributions to making our site a great resource for
```
Comment (jzb) [inline CONTRIBUTING.md]: Sentence fragment - for... developers and ops folks using containers?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,20 +1,40 @@
...
-# Project Atomic Site Style Guide
+# Contributing
+
+We want your contributions to making our site a great resource for 
+
+What follows is a guide to contributing content to the Project Atomic website.  
+
+## Content We Want
+
+We're looking for anything which relates to the various open source sub-projects of Atomic, or running Linux containers on Red Hat platforms.  In general, all content should relate to open source technologies.  This includes:
```
Comment (jzb) [inline CONTRIBUTING.md]: s/Red Hat platforms/CentOS, Fedora, or RHEL/

File: BUILD.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+   `./run-server.sh`
+
+   This will update your locally installed gems and start a Middleman
+   development server.
+
+2. Next, browse to <http://0.0.0.0:4567>
+
+3. Edit!
+
+   When you edit files (pages, layouts, CSS, etc.), the site will
```
Comment (bproffitt) [inline BUILD.md]: When you edit files (pages, layouts, CSS, etc.) and save them, the site will

**Comment by jberkus** [comment]
@jzb  None of the BUILD content is new, I just moved it from one file to another.  I felt that it made more sense for the README to be a "landing" page.

Can we merge this (with the corrections to the new content) and clean up the BUILD in a later commit?  That'll take a while because I need to test it all.  Frankly, I'm tempted to blow away all of the build instructions and just give instructions on how to build in a container, because really?

**Comment by mscherer** [comment]
I am ok to merge. For building in a container, I did had a branch of rhte original middleman repo somewhere, and I had to be "creative" to avoid either hitting cache issues while still using the cache. I can dig that later, or let you the fun of finding the solution (in my case, i was building a build container first using the md5 of gemfile.lock as a name, and then reusing that container to build the website).

File: BUILD.md (hunk 4 above)
Comment (jberkus) [inline BUILD.md]: Damn, why didn't anyone tell me about that?

**Comment by jberkus** [comment]
Updated per above.

I also added instructions on using docker.sh.

OK to merge?

**Pull request merged**